### DISCUSSION TITLE
Deploy to Digital Ocean

### DISCUSSION THREAD

**Discussion by LittleRichard** [mbround18/valheim-docker#339] [Show and tell]
I put this together to use with my server, it's a little janky in a couple spots and could be improved, but not bad for a weekend project!
https://github.com/LittleRichard/valheim-digitalocean

It combines this repo with a DigitalOcean API to start/stop cloud servers via scripts; this helps minimize server cost to just when you or your group are playing.  It's a bit technical, but IMO is approachable enough to be useful by anyone with a little bit of experience.  Hopefully other people here get some use out of it too, and if anyone wants to help streamline it that would be amazing!

**Comment by mbround18** [comment]
This is pretty awesome!! <3 i love it!
